docs/setup/config.md

```diff
@@ -602,26 +612,66 @@ You have to configure `jobmanager.archive.fs.dir` in order to archive terminated
 
 ## Background
 
+
 ### Configuring the Network Buffers
 
-If you ever see the Exception `java.io.IOException: Insufficient number of network buffers`, please use the following formula to adjust the number of network buffers:
+If you ever see the Exception `java.io.IOException: Insufficient number of network buffers`, you
```

Comment:
afaik those things only happen if you leave an empty space at the end of the line or so - here, the lines are properly joined inside HTML